hash_practice/exercises.py

```diff
@@ -2,18 +2,42 @@
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n)
+        Space Complexity: O(n)
     """

```

Comment:
👍 Time/space complexity is correct if the words are of limited length (like English words) and O(n * m log m) where m is the length of the words otherwise.